app/code/community/ShipStream/Sync/Model/Order/Shipment/Api.php

```diff
@@ -124,4 +131,104 @@ public function createWithTracking($orderIncrementId, $data)
 
         return $shipment->getIncrementId();
     }
+
+    /**
+     * Retrieve Shipped Order Item Qty from Shipstream shipment packages
+     * @param $order
+     * @param $data
+     * @return array
+     */
+    protected function _getShippedItemsQty($order, $data): array
+    {
+        $orderItems = [];
+        $itemShippedQty = [];
+
+        //get Order Item Ids from magento order items
+        $orderItemsData = $order->getAllItems();
+        foreach ($orderItemsData as $orderItem){
+            $orderItems[$orderItem->getSku()] = $orderItem->getItemId();
+
+        }
+
+        //payload data to create shipment in openmage for only items shipped from shipstream
+        foreach ($data['packages'] as $package) {
+            foreach ($package['items'] as $item){
+                $key = $orderItems[$item['sku']];
+                if(isset($itemShippedQty[$key])) {
+                    $itemShippedQty[$key] = bcadd($itemShippedQty[$key], $item['order_item_qty'],4);
+                }
+                else {
+                    $itemShippedQty[$key] = floatval($item['order_item_qty']);
```

Comment:
Not necessary to use floatval when bcmath is used.
```suggestion
                    $itemShippedQty[$key] = $item['order_item_qty'];
```